packages/material-ui-codemod/README.md

```diff
@@ -69,53 +69,74 @@ npx @material-ui/codemod@next v5.0.0/preset-safe <path|folder>
 
 The list includes these transformers
 
-- [`adapter-v4`](#adapter-v4)
-- [`autocomplete-rename-closeicon`](#autocomplete-rename-closeicon)
-- [`autocomplete-rename-option`](#autocomplete-rename-option)
-- [`avatar-circle-circular`](#avatar-circle-circular)
-- [`badge-overlap-value`](#badge-overlap-value)
-- [`box-borderradius-values`](#box-borderradius-values)
-- [`box-rename-css`](#box-rename-css)
-- [`box-rename-gap`](#box-rename-gap)
-- [`button-color-prop`](#button-color-prop)
-- [`chip-variant-prop`](#chip-variant-prop)
-- [`circularprogress-variant`](#circularprogress-variant)
-- [`collapse-rename-collapsedheight`](#collapse-rename-collapsedheight)
-- [`core-styles-import`](#core-styles-import)
-- [`create-theme`](#create-theme)
-- [`dialog-props`](#dialog-props)
-- [`dialog-title-props`](#dialog-title-props)
-- [`emotion-prepend-cache`](#emotion-prepend-cache)
-- [`expansion-panel-component`](#expansion-panel-component)
-- [`fab-variant`](#fab-variant)
-- [`fade-rename-alpha`](#fade-rename-alpha)
-- [`grid-justify-justifycontent`](#grid-justify-justifycontent)
-- [`grid-list-component`](#grid-list-component)
-- [`icon-button-size`](#icon-button-size)
-- [`material-ui-styles`](#material-ui-styles)
-- [`material-ui-types`](#material-ui-types)
-- [`modal-props`](#modal-props)
-- [`moved-lab-modules`](#moved-lab-modules)
-- [`pagination-round-circular`](#pagination-round-circular)
-- [`root-ref`](#root-ref)
-- [`skeleton-variant`](#skeleton-variant)
-- [`styled-engine-provider`](#styled-engine-provider)
-- [`table-props`](#table-props)
-- [`tabs-scroll-buttons`](#tabs-scroll-buttons)
-- [`textarea-minmax-rows`](#textarea-minmax-rows)
-- [`theme-augment`](#theme-augment)
-- [`theme-breakpoints`](#theme-breakpoints)
-- [`theme-breakpoints-width`](#theme-breakpoints-width)
-- [`theme-options`](#theme-options)
-- [`theme-palette-mode`](#theme-palette-mode)
-- [`theme-provider`](#theme-provider)
-- [`theme-spacing`](#theme-spacing)
-- [`theme-typography-round`](#theme-typography-round)
-- [`transitions`](#transitions)
-- [`use-autocomplete`](#use-autocomplete)
-- [`use-transitionprops`](#use-transitionprops)
-- [`with-mobile-dialog`](#with-mobile-dialog)
-- [`with-width`](#with-width)
+- [@material-ui/codemod](#material-uicodemod)
+  - [Setup & run](#setup--run)
+    - [jscodeshift options](#jscodeshift-options)
+    - [Recast Options](#recast-options)
+  - [Included scripts](#included-scripts)
```

Comment:
This change shouldn't have been in, it's something we already fixed in the past. AFAIK, to revert

Reply:
🥲 my auto format, fixed in #27404